wizeline/wize-q-remix
# feat(frontend): add search questions component to navbar

#### What does this PR do?
- Implements new search component into the Remix project
- Adds lodash to the project
- Uses lodash.debounce and React useCallback to call questions search query on intervals to avoid performance issues.
- Adds global eslint disable for exhaustive deps, as a lot of components currently require this. 

#### Where should the reviewer start?
- See the new additions to the SearchBar and Navbar components
- Install new Lodash package with `npm install`
 
#### How should this be manually tested?
1. Go to any page
2. Type on the navbar search bar
3. Verify questions listed have the text that was typed
4. Click on a question and verify it redirects to the question detail page
5. :tada:

#### What are the relevant tickets?
Closes #26 

#### Screenshots
**Note**: 404 are shown in question detail as this is currently in WIP #30 

https://user-images.githubusercontent.com/38927620/195680728-0566c17b-4d44-4883-8596-febae63d11fc.mov



https://user-images.githubusercontent.com/38927620/195680756-84efab80-8161-483a-9dfe-57637e762acb.mov
